build.gradle

```diff
@@ -109,13 +109,26 @@ thirdPartyAudit.enabled = false
 def es_tmp_dir = rootProject.file('build/private/es_tmp').absoluteFile
 es_tmp_dir.mkdirs()
 
+task cmakeJniLib(type:Exec) {
+    workingDir 'jni'
+    commandLine 'cmake', '.'
+}
+
+task buildJniLib(type:Exec) {
+    dependsOn cmakeJniLib
+    workingDir 'jni'
+    commandLine 'make'
+}
+
```

Comment:
why not club this to one task

task cmakeJniLib(type:Exec) {
    workingDir 'jni'
    commandLine 'cmake', '.'
     commandLine 'make'
}

Reply:
The following syntax does not work. `commandLine 'cmake', '.'` is overridden with `commandLine 'make'`. 
```
task buildJniLib(type:Exec) {
workingDir 'jni'
commandLine 'cmake', '.'
commandLine 'make'
}
```

Additionally, having 2 separate tasks allows a user to just run the cmake task if they want.